File: 268-missing-number/missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n= nums.size();
        int answer=0;
        
        vector<int>::iterator itr;
       
for(int i=0;i<=n;i++){
    itr=find(nums.begin(),nums.end(),i);
    if(itr==nums.end()){
        answer=i;
        break;
    }
}
        
        return answer;
    }
};

File: 268-missing-number/missing-number.go
func missingNumber(nums []int) int {
    number:=map[int]bool{}
    for _,num:=range nums{
        number[num]=true
    }
    for i:=0;i<=len(nums);i++{
        if !number[i]{
            return i
        }
    }
    
    return -1
}